include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -527,6 +527,29 @@ typedef struct OrtApiBase OrtApiBase;
 */
 ORT_EXPORT const OrtApiBase* ORT_API_CALL OrtGetApiBase(void) NO_EXCEPTION;
 
+/** \brief Thread work loop function
+*
+* Onnxruntime will provide the working loop on custom thread creation
+* Argument is an onnxruntime built-in type which will be provided when thread pool calls CustomCreateThreadFn
+*/
+typedef void (*OrtThreadWorkerFn)(void* worker_fn_param);
+
+#define CUSTOM_THREAD_HANDLE void*
+
+/** \brief Custom thread creation function
+*
+* The function should return a thread handle to be used in onnxruntime thread pools
+* Onnxruntime will throw exception on return value of nullptr or 0, indicating that the function failed to create a thread
+*/
+typedef CUSTOM_THREAD_HANDLE (*CustomCreateThreadFn)(void* custom_thread_creation_options, OrtThreadWorkerFn ort_thread_worker_fn, void* worker_fn_param);
```

Comment:
To follow naming convention: OrtCreateCustomThreadFn

Reply:
Done.